[2023-08-18T23:10:46.879+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian permission to 509
[2023-08-18T23:10:46.880+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian'
[2023-08-18T23:10:46.929+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian permission to 509
[2023-08-18T23:10:46.933+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian'
[2023-08-18T23:10:47.204+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_bash_select_dog.t2_pomeranian manual__2023-08-18T23:10:41.081828+00:00 [queued]>
[2023-08-18T23:10:47.217+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_bash_select_dog.t2_pomeranian manual__2023-08-18T23:10:41.081828+00:00 [queued]>
[2023-08-18T23:10:47.218+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-18T23:10:47.257+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): t2_pomeranian> on 2023-08-18 23:10:41.081828+00:00
[2023-08-18T23:10:47.266+0000] {standard_task_runner.py:57} INFO - Started process 362 to run task
[2023-08-18T23:10:47.281+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_bash_select_dog', 't2_pomeranian', 'manual__2023-08-18T23:10:41.081828+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dags_bash_select_dog.py', '--cfg-path', '/tmp/tmp5a40k4f5']
[2023-08-18T23:10:47.291+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask t2_pomeranian
[2023-08-18T23:10:47.492+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian permission to 509
[2023-08-18T23:10:47.495+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_select_dog/run_id=manual__2023-08-18T23:10:41.081828+00:00/task_id=t2_pomeranian'
[2023-08-18T23:10:47.505+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_bash_select_dog.t2_pomeranian manual__2023-08-18T23:10:41.081828+00:00 [running]> on host 7ff4ca19a8ff
[2023-08-18T23:10:47.730+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_bash_select_dog' AIRFLOW_CTX_TASK_ID='t2_pomeranian' AIRFLOW_CTX_EXECUTION_DATE='2023-08-18T23:10:41.081828+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-18T23:10:41.081828+00:00'
[2023-08-18T23:10:47.732+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-18T23:10:47.733+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '/opt/***/plugins/shell/select_dog.sh POMERANIAN']
[2023-08-18T23:10:47.742+0000] {subprocess.py:86} INFO - Output:
[2023-08-18T23:10:47.755+0000] {subprocess.py:93} INFO - You selected POMERANIAN!
[2023-08-18T23:10:47.756+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-18T23:10:47.801+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dags_bash_select_dog, task_id=t2_pomeranian, execution_date=20230818T231041, start_date=20230818T231047, end_date=20230818T231047
[2023-08-18T23:10:47.892+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-18T23:10:47.910+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
